Sources/CryptoService/Signing/CryptoSigningService.swift

```diff
@@ -0,0 +1,83 @@
+import BigInt
+import CryptoKit
+import Foundation
+import Security
+
+public enum SigningServiceError: Error {
+    /// The public key external representation could not be created
+    case couldNotCreatePublicKeyAsData
+    
+    /// The did key external representation could not be created
+    case couldNotCreateDIDKeyAsData
+    
+    /// No result was returned but no error was thrown creating the signature by the `Security` framework
+    case unknownCreateSignatureError
+}
+
+public final class CryptoSigningService: SigningService {
+    private let keyStore: KeyStore
+    
+    /// The public key in either raw or did:key format, as defined in the w3c specification.
+    /// https://w3c-ccg.github.io/did-method-key/
+    public func publicKey(didKey: Bool = false) throws -> Data {
```

Comment:
should this be an enum with supported formats? It gives us more flexibility to extend the API in the future.
```
enum KeyFormat {
    case decentralisedIdentifier
    case jwks
}
```